pkg/features/kube_features.go

```diff
@@ -826,6 +826,14 @@ const (
 	//
 	// Stop auto-generation of secret-based service account tokens.
 	LegacyServiceAccountTokenNoAutoGeneration featuregate.Feature = "LegacyServiceAccountTokenNoAutoGeneration"
+
+	// owner: @xing-yang
+	// kep: http://kep.k8s.io/2268
+	// alpha: v1.24
+	//
+	// Allow pods to failover to a different node in case of non graceful node shutdown
+	// before the 6 min timeour period
```

Comment:
I think we can remove this "before the 6 min timeour period" here as it is not clear where this timeout happens with this text here.